### PR TITLE
feat(documentation): add docker content

### LINKED ISSUE #224: Docker & Docker Compose Tutorial
Describe docker docker compise and kubernetes

What is a container

### PR DESCRIPTION
**Pull request by leisterfrancisco**
### GH Issue

Resolves #224 

### Steps to test
1. Check needed content is included

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

**Comment by xavier506** [comment]
Looks Good! 

Perhaps we can find a diagram to help visualize some of the concepts better? 

For example :

image

image

(lets make sure that the images are free to use)

Also should we add a link to the devops page somewhere?

**Comment by leisterfrancisco** [comment]
> 
> 
> Looks Good!
> 
> Perhaps we can find a diagram to help visualize some of the concepts better?
> 
> For example :
> 
> image
> 
> image
> 
> (lets make sure that the images are free to use)
> 
> Also should we add a link to the devops page somewhere?

Good, working on

File: sidebars.js [hunk 1]
```diff
@@ -60,6 +60,7 @@ module.exports = {
         'tools/nodeos-command-line-reference',
         'tools/block-explorers',
         'tools/eosio-sdk-libraries',
+        'tools/docker',
```
Comment (xavier506) [inline sidebars.js:63]: under tools or tutorials?

Comment (leisterfrancisco) [reply sidebars.js:63]: tutorials

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by Edgar097 — APPROVED** [review]

**Review by xavier506 — APPROVED** [review]

File: docs/tutorials/docker.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+id: docker
+title: Docker
+sidebar_label: Docker
+---
+
+Docker is a tool that allows you to quickly create, test, and deploy applications. The idea behind Docker is to create “containers” that are lightweight and portable so that applications can run on any machine with Docker installed, regardless of the operating system.
+
+For example, each node of the blockchain network or service / application developed will be compiled into a docker image that contains all of its dependencies and is deployed to one of several infrastructure providers with docker available.
+
+![Docker Structure](https://upload.wikimedia.org/wikipedia/commons/thumb/0/0a/Docker-containerized-and-vm-transparent-bg.png/800px-Docker-containerized-and-vm-transparent-bg.png)
```
Comment (xavier506) [inline docs/tutorials/docker.md:11]: should we host this image ourselves?

Comment (leisterfrancisco) [reply docs/tutorials/docker.md:11]: Good, I going to do it

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**